_posts/2024-03-01-pyos-newsletter-march-2024.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: single
```

Comment:
this is great - have a look at this page
https://www.pyopensci.org/blog/scipy-2023-community.html
it uses the classes:wide option to make for a wider layout. i wonder if what we've been using is too narrow?

what do you think? im only bringing this up as we can establish our "blog" style now that you are working on this!! and i think what's bothering me is the font size / characters per line vs width of the div/ container.

Reply:
@lwasser I think I prefer the page with `classes: wide` and will put that in for this post! I think there's still room to figure out the best visual look, but this is a good stepping stone.

Reply:
perfect. me too! i do agree the visual look isn't quite there but atleast that layout gives us more space. i've been using a custom wide layout for most of the none blog pages. we can always revise a template.